ci/scripts/sandbox-deploy.sh

```diff
@@ -14,6 +14,8 @@ echo "Using AWS_REGION: ${AWS_REGION}"
 export ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
 
 # Build
-./ci/scripts/build.sh
+npm run build
```

Comment:
```suggestion
./ci/scripts/build.sh
```

Let's keep a consistent CI structure

Reply:
in wayfinderbackend and wayfinder-slack-api it is 'npm run build'

Made this change to be in alignment with the other repos